git_machete/utils.py

```diff
@@ -94,7 +94,21 @@ def debug(msg: Optional[str] = None) -> None:
     if debug_mode:
         function_name = bold(inspect.stack()[1].function)
         args, _, _, values = inspect.getargvalues(inspect.stack()[1].frame)
-        function_args = bold(f"({', '.join([arg+'='+str(values[arg]) if arg != 'self' else 'self' for arg in args])})")
+
+        redact_arg_by_name = ['token']
+        for arg in redact_arg_by_name:
+            values[arg] = '***'
+
+        redact_arg_by_value = ['ghp_']
```

Comment:
That's the only prefix used by GitHub tokens right?

Reply:
```suggestion
        redacted_arg_by_prefix = {'ghp_'}
```

Reply:
Yeah, I suppose only personal access tokens can reasonably be used by git-machete... but let's include all 5 prefixes for the sake of forgetting about this problem once and for all